posts/GSoC_2019_Fourth_Blog.md

```diff
@@ -0,0 +1,74 @@
+---
+title: GSoC 2019 Third Blog
+date: 2019-07-20 15:03 UTC-19:03
+slug: GSoC_2019_Fourth_Blog
+tags: GSoC 2019
+description: 2019 Google Summer of Code Cantera Project Blog
+type: text
+author: Chao Xu
+---
+
+# GSoC 2019 Fourth Blog
+
+After two months, the functions of ChemCheck are comparatively robust. We improved editing page and ck2yaml error reporting page this week, and we
+added pages for user accounts management.
+
+<!-- TEASER_END -->
+
+We have included download function for the editing page, which users can download the edited file to their local directory. It will more convenient
+if changes can be saved to the original file online, so that users do not have to reupload their files, but we will leave it for now since an
+editing function is not the most important part. The current exception handling for editing page is striping out all the special signs which cannot
+be recognized by ace-editor, and it could also be improved by implementing some python library to guess encoding in the future. 
```

Comment:
```suggestion
be recognized by ace-editor, and it could also be improved by implementing some Python library to guess the encoding of the file in the future. 
```